.gitignore

```diff
@@ -1,2 +1,181 @@
-# Emacs backup files
-*~
```

Comment:
Really nice additions and aligning this with molecule / lint. Yet, these two original lines above should still stay there. The rule refer to temp files that indeed should be ignored. After this minor edit I am happy to merge.